tensorflow/windows.bzl

```diff
@@ -0,0 +1,84 @@
+  "//tensorflow/core/platform/cloud:http_request_test",
+  "//tensorflow/core/platform/cloud:google_auth_provider_test",
+  "//tensorflow/core/platform/cloud:gcs_file_system_test",
+  "//tensorflow/core/kernels/cloud:bigquery_table_accessor_test",
+  "//tensorflow/core/kernels/hexagon:quantized_matmul_op_for_hexagon_test",
+  "//tensorflow/core/kernels:sparse_add_op_test",
+  "//tensorflow/core/kernels:spacetobatch_benchmark_test_gpu",
+  "//tensorflow/core/kernels:spacetobatch_benchmark_test",
+  "//tensorflow/core/kernels:requantize_op_test",
+  "//tensorflow/core/kernels:requantization_range_op_test",
+  "//tensorflow/core/kernels:quantized_reshape_op_test",
+  "//tensorflow/core/kernels:quantized_pooling_ops_test",
+  "//tensorflow/core/kernels:quantized_matmul_op_test",
+  "//tensorflow/core/kernels:quantized_conv_ops_test",
+  "//tensorflow/core/kernels:quantized_concat_op_test",
+  "//tensorflow/core/kernels:quantized_bias_add_op_test",
+  "//tensorflow/core/kernels:quantized_batch_norm_op_test",
+  "//tensorflow/core/kernels:quantized_activation_ops_test",
+  "//tensorflow/core/kernels:quantize_op_test",
+  "//tensorflow/core/kernels:quantize_down_and_shrink_range_op_test",
+  "//tensorflow/core/kernels:quantize_and_dequantize_op_test_gpu",
+  "//tensorflow/core/kernels:quantize_and_dequantize_op_test",
+  "//tensorflow/core/kernels:quantization_utils_test",
+  "//tensorflow/core/kernels:debug_ops_test",
+  "//tensorflow/core/kernels:control_flow_ops_test",
+  "//tensorflow/core/kernels:cast_op_test_gpu",
+  "//tensorflow/core/kernels:cast_op_test",
+  "//tensorflow/core/distributed_runtime/rpc:rpc_rendezvous_mgr_test_gpu",
+  "//tensorflow/core/distributed_runtime/rpc:rpc_rendezvous_mgr_test",
+  "//tensorflow/core/distributed_runtime/rpc:grpc_tensor_coding_test",
+  "//tensorflow/core/distributed_runtime/rpc:grpc_session_test_gpu",
+  "//tensorflow/core/distributed_runtime/rpc:grpc_session_test",
+  "//tensorflow/core/distributed_runtime/rpc:grpc_channel_test_gpu",
+  "//tensorflow/core/distributed_runtime/rpc:grpc_channel_test",
+  "//tensorflow/core/distributed_runtime:remote_device_test_gpu",
+  "//tensorflow/core/distributed_runtime:remote_device_test",
+  "//tensorflow/core/distributed_runtime:executor_test_gpu",
+  "//tensorflow/core/distributed_runtime:executor_test",
+  "//tensorflow/core/debug:debug_gateway_test",
+  "//tensorflow/core/debug:debug_grpc_io_utils_test",
+  "//tensorflow/core:util_reporter_test",
+  "//tensorflow/core:util_memmapped_file_system_test",
+  "//tensorflow/core:platform_subprocess_test",
+  "//tensorflow/core:platform_profile_utils_cpu_utils_test",
+  "//tensorflow/core:platform_port_test",
+  "//tensorflow/core:lib_strings_strcat_test",
+  "//tensorflow/core:lib_jpeg_jpeg_mem_unittest",
+  "//tensorflow/core:lib_core_notification_test",
+  "//tensorflow/core:framework_partial_tensor_shape_test",
+  "//tensorflow/core/debug:debug_io_utils_test",
+]
+
+def if_fail_on_windows(test_name):
```

Comment:
I have bad feelings about this construct, but I am not sure how to do things better.
Maybe we can have the list live in a bash or a python file, and not use bazel query for constructing the test list to run?
Or maybe even manually exclude them directly in bazel test run?